.travis.yml

```diff
@@ -33,6 +33,13 @@ jobs:
     - <<: *test-template
       name: Full Test on Linux
       os: linux
+      dist: xenial
+      before_install:
```

Comment:
Could you put this link: https://github.com/containers/buildah/blob/master/install.md a line above to show where we have taken these commands from?